DateRangeBox: Add Getting Started with DateRangeBox guide (#5041)
DevExpress/devextreme-documentation: concepts/05 UI Components/DateRangeBox/00 Getting Started with DateRangeBox/10 Set the Accepted Date Range.md

> You can define constraints for date ranges that can be selected by end-users. Specify the [min](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#max) properties.

Reviewer: ```suggestion
You can define constraints for date ranges that can be selected by end users. Specify the [min](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#max) properties.
```

> You can define an available range from which end-users can select dates. For this purpose, specify the [min](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#max) properties.

Reviewer 2: ```suggestion
You can define constraints for date ranges that can be selected by end-users. Specify the [min](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#max) properties.
```

            }
        }
    }
    </script>

    <!-- tab: App.vue (Composition API) -->
    <template>
        <DxDateRangeBox ...
            :min="minDate"
            :max="maxDate"
        />
    </template>

    <script setup>
    import 'devextreme/dist/css/dx.light.css';

    import { DxDateRangeBox } from 'devextreme-vue/date-range-box';

    const minDate = new Date(new Date().setDate(new Date().getDate() - 14)), // 2 weeks before the current date
    const maxDate = new Date(new Date().setDate(new Date().getDate() + 14)), // 2 weeks after the current date
    </script>

##### React

    <!-- tab: App.js -->
    import React from 'react';

    import 'devextreme/dist/css/dx.light.css';

    import { DateRangeBox } from 'devextreme-react/date-range-box';

    const minDate = new Date(new Date().setDate(new Date().getDate() - 14)), // 2 weeks before the current date
    const maxDate = new Date(new Date().setDate(new Date().getDate() + 14)), // 2 weeks after the current date

    export default function App() { 
        return (
            <DateRangeBox ...
                min={minDate}
                max={maxDate}
            />
        );
    }

---

> Run the code and ensure that the only available dates are between two weeks before and two weeks after the current date. Next, we will configure the UI component's initial values.

Reviewer 2: ```suggestion
Run the code. You can now select dates only within the predefined range: from two weeks before the current date to two weeks after. The next step specifies the date range selected at startup.
```

You can define an available range from which end-users can select dates. For this purpose, specify the [min](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxDateRangeBox/Configuration/#max) properties:

---
##### jQuery

    <!-- tab: index.js -->
    $(function() {
        $("#dateRangeBox").dxDateRangeBox({

>             min: new Date(1920, 0, 1),

Reviewer 3: may be we should set more strict man/max? With values like this user won't see how disabling dates less than min/bigger than max working. How about +- two weeks from current date?

Author: Ok, I think it is good idea too
